org.desmume.DeSmuME.appdata.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.desmume.DeSmuME</id>
+  <name>DeSmuME</name>
+  <summary>Nintendo DS emulator</summary>
+  <project_license>GPL-2.0-or-later</project_license>
+  <metadata_license>CC0-1.0</metadata_license>
+  <developer_name>DeSmuME Team</developer_name>
+  <icon type="remote">https://linkmauve.fr/files/desmume-logo.svg</icon>
+
+  <url type="homepage">http://desmume.org</url>
+  <url type="bugtracker">https://github.com/TASVideos/desmume/issues</url>
+
+  <description>
+    <p>
+      DeSmuME is a Nintendo DS emulator.
+    </p>
+  </description>
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://linkmauve.fr/files/desmume-pokémon.png</image>
```

Comment:
This screenshot of DeSmuME at its native resolution is smaller than the recommended size, but I’d like to keep it that way instead of padding it with useless blank pixels, will that be an issue?